feat/gene/gene_test.go

```diff
@@ -233,6 +242,26 @@ var transcriptTests = []struct {
 		ExonicLen: 1200,
 	},
 	{
```

Comment:
I'd like to see another test where a transcript is on a gene on a contig on a supercontig on an ultra contig.